unittest_requirements.txt

```diff
@@ -0,0 +1 @@
+mock
```

Comment:
Why `mock` is required?
Starting from python 3.3 this package is included into standard library as `unittest.mock`.

Reply:
it came from template:
https://github.com/pulp/plugin_template/blob/master/templates/test/unittest_requirements.txt.j2
I'll remove it there